docs/UserGuide.md

```diff
@@ -3,271 +3,443 @@ layout: page
  e.g. if the command specifies `help 123`, it will be interpreted as `help`.
 
-
 </div>
 
+## Feature Description
+
 ### Viewing help : `help`
 
-Shows a message explaining how to access the help page.
+**Purpose**: Shows a message explaining how to access this help page.
 
-![help message](images/helpMessage.png)
+![help message](images/help-message.png)
 
-Format: `help`
+**Format**: `help`
 
 
-### Adding a person: `add`
+### Add client contact: `add`
 
-Adds a client to ClientBook.
+**Purpose**: Adds a client as contact to ClientBook.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [i/POLICY_ID[>POLICY_URL]] [t/TAG]…​`
+**Format**: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [i/POLICY_ID[>POLICY_URL]] [t/TAG]…​`
 
-<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+:bulb: 
+**Tip**:
 A person can have any number of tags and insurance policies (including 0)
-</div>
+
 
 * It is perfectly fine to not include the URL to the insurance policy document!
 * To include the URL, remember to use '>' to indicate that a particular insurance policy is linked to a URL, as shown in the second example below.
 
-Examples:
-* (example of a client with insurance policy but no URL)`add n/John Doe p/98765432 e/johnd@example.com a/Orchard i/Policy_1273 t/basic`
-* (example of a client with insurance policy and URL)`add n/Tom Tan p/91234567 e/tomt@example.com a/Orchard i/Policy_1274>www.myinsurancecompany.com/policy_1274 t/basic`
-* (example with no insurance policy and no tag)`add n/Betsy Crowe e/betsycrowe@example.com a/Kent Ridge`
+**Examples**:
+* Example of a client with insurance policy but no URL associated with insurance policy
+    * `add n/John Doe p/98765432 e/johnd@email.com a/Orchard i/Pol_#123456 t/basic`
+* Example of a client with insurance policy and URL associated with insurance policy
+    * `add n/Tom Tan p/91234567 e/tomt@email.com a/Orchard i/Policy_1274>www.myinsurancecompany.com/policy_1274 t/basic`
+* Example with no insurance policy and no tag
+    * `add n/Betsy Crowe e/betsycrowe@email.com a/Kent Ridge`
 
-### Listing all persons : `list [-ATTRIBUTE]`
 
-![list_with_filter](images/list_with_filter.png)
+### Edit client contact: `edit`
 
-Shows a list of all clients in ClientBook. An optional attribute option can be added to show a list with only the specified attribute.
+**Purpose**: Edits an existing client contact in the ClientBook.
 
-Format: `list [-ATTRIBUTE]…​`
+**Format**: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [i/POLICY_NUMBER] [t/TAG]…​`
 
-Examples:
-* `list` Shows a list of all clients and all their information
-* `list -policy` Shows a list of all clients and their policy numbers
-* `list -phone -policy` Shows a list of all clients, their phone number, and their policy numbers
+* Edits the client at the specified `INDEX`.
+    * The index refers to the index number shown in the displayed client list.
+    * The index must be a positive integer 1, 2, 3, …​
```

Comment:
Index has to be within the range of the list as well.

Reply:
Okay, I changed the positive integer line to the following: 
* The index must be more than 1, and less than or equal to the index of the last item in the displayed list.

Reply:
Yeah, this is much clearer!